ligo/stablecoin/spec.ligo

```diff
@@ -79,8 +79,7 @@ end
 type balance_of_params is michelson_pair_right_comb(balance_of_params_)
 
 type token_metadata_ is record
-  token_id  : token_id
```

Comment:
Why did you remove `token_id` from here? AFAIU it contradicts FA2. We even have discussed it with Sandeep in [this thread](https://serokell.slack.com/archives/CUMAQ3PL4/p1592391873179000) before.